assets/tiled_class_property.tmx

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<map version="1.5" tiledversion="1.7.2" orientation="orthogonal" renderorder="right-down" width="2" height="2" tilewidth="32" tileheight="32" infinite="0" nextlayerid="3" nextobjectid="4">
```

Comment:
Wasn't `class` added on Tiled 1.8? How was this file made?

Reply:
I modified the tiled_object_property.tmx using values from a map file of my own. Generating a full file for testing through Tiled would probably have been cleaner.

Reply:
I guess we can leave it as-is now, since it's not relevant for the test.